### DISCUSSION TITLE
Help with Readme

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21465] [Other Feature Feedback, Questions, & Ideas]
<p>Hello…<br>
I want to make a Readme file better with adding things like Buttons and cool images, would I need to use HTML And CSS and Javascript for backend or can I just use MD If so how?<br>
James</p>

**Comment by mpboom** [comment, marked as answer]
<p>Markdown (or GFM) has no native support for any buttons, but has support for images. I believe you can put HTML in your Markdown, at least certain tags seem to work well. <a href="https://github.com/Ileriayo/markdown-badges" rel="noopener nofollow ugc">You can get creative with them</a>, a quick Google search reveals.</p>
